server/schedule/operator_controller.go

```diff
@@ -741,17 +736,17 @@ func (oc *OperatorController) getOrCreateStoreLimit(storeID uint64) *ratelimit.B
 }
 
 // GetAllStoresLimit is used to get limit of all stores.
-func (oc *OperatorController) GetAllStoresLimit() map[uint64]float64 {
+func (oc *OperatorController) GetAllStoresLimit() map[uint64]*StoreLimit {
 	oc.RLock()
 	defer oc.RUnlock()
-	ret := make(map[uint64]float64)
+	limits := make(map[uint64]*StoreLimit)
 	for storeID, limit := range oc.storesLimit {
 		store := oc.cluster.GetStore(storeID)
 		if !store.IsTombstone() {
-			ret[storeID] = limit.Rate() / float64(operator.RegionInfluence)
```

Comment:
why to remove `float64(operator.RegionInfluence)`?

Reply:
It is not removed, it is assigned with the new `limit` witch contains the `rate`

Reply:
do you mean `NewStoreLimit`?

Reply:
`float64(operator.RegionInfluence)` will be used later, it is not removed, it is just used here, this is because I want to return the `limit` object not only the `rate` itself.

Reply:
Just search `float64(operator.RegionInfluence)` on this page and you will get the whole changes.